core/bin/snapshot_creator/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
```

Comment:
drive-by comment:

If we are adding a new crate/binary - please also create a README.md file describing what this tool is for, with some examples on how to run it.